www/js/search.js

```diff
@@ -671,6 +671,10 @@ module.exports = {
             h('i.fa.fa-fw.fa-home'),
             ' ',
             item.Gurmukhi,
+            h(
+              'span.hidden.main-letters',
+              item.MainLetters,
+            ),
```

Comment:
Could this be a data attribute instead of adding them to the DOM? That way you can avoid a costly `querySelector` and just get the value from the attribute itself.